src/components/form/FormInviteFriends.vue

```diff
@@ -0,0 +1,204 @@
+ * triggered only when component’s validate() method is manually called or
+ * when the wrapper QForm submits itself.
+ *
+ * @example
+ * <form-invite-friends />
+ *
+ * @see [Figma Design](https://www.figma.com/file/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?type=design&node-id=4858%3A105087&mode=dev)
+ */
+
+// libraries
+import { computed, defineComponent, ref } from 'vue';
+
+// composables
+import { i18n } from 'src/boot/i18n';
+import { useValidation } from 'src/composables/useValidation';
+
+// types
+import type { FormOption } from '../types/Form';
+
+export default defineComponent({
+  name: 'FormInviteFriends',
+  setup() {
+    const emailAddresses = ref<string>('');
+    const language = ref<string>(i18n.global.locale);
+
+    // Flag icons
+    const iconFlagCz = `img:${
+      new URL('../../assets/svg/flag-cz.svg', import.meta.url).href
+    }`;
+    const iconFlagSk = `img:${
+      new URL('../../assets/svg/flag-sk.svg', import.meta.url).href
+    }`;
+    const iconFlagEn = `img:${
+      new URL('../../assets/svg/flag-en.svg', import.meta.url).href
+    }`;
+    // Country options
+    const optionsLanguage: FormOption[] = [
+      {
+        label: 'Czech',
+        value: 'cs',
+        icon: iconFlagCz,
+      },
+      {
+        label: 'English',
+        value: 'en',
+        icon: iconFlagEn,
+      },
+      {
+        label: 'Slovak',
+        value: 'sk',
+        icon: iconFlagSk,
+      },
+    ];
+    const selectedLanguage = computed((): FormOption | null => {
+      const languageObject: FormOption | undefined = optionsLanguage.find(
+        (option) => option.value === language?.value,
+      );
+      if (languageObject) {
+        return languageObject;
+      }
+      return null;
+    });
+
+    const { isFilled } = useValidation();
+
+    return {
+      emailAddresses,
+      iconFlagCz,
+      iconFlagSk,
+      iconFlagEn,
+      language,
+      optionsLanguage,
+      selectedLanguage,
+      isFilled,
+    };
+  },
+});
+</script>
+
+<template>
+  <div class="row q-col-gutter-md" data-cy="form-invite-friends">
+    <!-- Description text -->
+    <div class="col-12 col-sm-6" data-cy="column-1">
+      <div
+        v-html="$t('onboarding.descriptionInvite')"
+        class="text-grey-10 q-mb-lg"
+        data-cy="invite-description"
+      />
+      <!-- Message input widget -->
+      <div class="q-my-md" data-cy="invite-email-addresses">
+        <label
+          for="invite-email-addresses"
+          class="text-caption text-bold text-grey-10"
+        >
+          {{ $t('onboarding.labelInviteEmailAddresses') }}
+        </label>
+        <q-input
+          dense
+          outlined
+          hide-bottom-space
+          lazy-rules="ondemand"
+          v-model="emailAddresses"
+          id="invite-email-addresses"
+          name="email-addresses"
+          type="textarea"
+          :rules="[
+            (val) => isFilled(val) || $t('onboarding.messageRequiredAddresses'),
```

Comment:
You need here much more advance validation rules here.

1. Checking whether individual email addresses  are separated by a comma
2. Checking whether individual email addresses have the correct format

Reply:
Extended the validation method (using the function isEmail).